plugins/dicom_viewer/server/__init__.py

```diff
@@ -195,9 +197,46 @@ def _uploadHandler(event):
     events.trigger('dicom_viewer.upload.success')
 
 
+def dicomSubstringSearchHandler(query, types, user=None, level=None, limit=0, offset=0):
+    """
+    Provide a substring search on both keys and values.
+    """
+    if types != ['item']:
+        raise RestException('The dicom search is only able to search in Item.')
+
+    # The insensitive Case is not perfect
+    # We will need to search into object value as improvment.
+    jsQuery = """
+        function() {
+            const key = '%(key)s'.toLowerCase();
```

Comment:
`const` is ES6 syntax. For compatibility with older versions of MongoDB, we should only use ES5 syntax, so this should be `var`.

Reply:
Also, can you name this `queryKey` and `queryValue`? In addition to being clearer, this will fix a bug in the current implementation.